--- src/async_client.cpp.old	2020-11-13 12:13:30.694805240 -0700
+++ src/async_client.cpp	2020-11-13 12:15:40.033041375 -0700
@@ -35,25 +35,6 @@
 #endif
 
 /////////////////////////////////////////////////////////////////////////////
-// Paho C logger
-
-enum LOG_LEVELS {
-	INVALID_LEVEL = -1,
-	TRACE_MAX = 1,
-	TRACE_MED,
-	TRACE_MIN,
-	TRACE_PROTOCOL,
-	LOG_PROTOCOL = TRACE_PROTOCOL,
-	LOG_ERROR,
-	LOG_SEVERE,
-	LOG_FATAL,
-};
-
-extern "C" {
-	void Log(enum LOG_LEVELS, int, const char *, ...);
-}
-
-/////////////////////////////////////////////////////////////////////////////
 
 namespace mqtt {
 
@@ -150,8 +131,6 @@
 // is notified of success on a normal connect with callbacks. 
 void async_client::on_connected(void* context, char* cause)
 {
-	::Log(TRACE_MIN, -1, "[cpp] on_connected");
-
 	if (context) {
 		async_client* cli = static_cast<async_client*>(context);
 		callback* cb = cli->userCallback_;
@@ -172,8 +151,6 @@
 // MQTTAsync_setCallbacks(). 
 void async_client::on_connection_lost(void *context, char *cause)
 {
-	::Log(TRACE_MIN, -1, "[cpp] on_connection lost");
-
 	if (context) {
 		async_client* cli = static_cast<async_client*>(context);
 		callback* cb = cli->userCallback_;
--- src/token.cpp.old	2020-11-13 12:14:59.155612309 -0700
+++ src/token.cpp	2020-11-13 12:16:09.034371509 -0700
@@ -24,25 +24,6 @@
 #include <iostream>
 
 /////////////////////////////////////////////////////////////////////////////
-// Paho C logger
-
-enum LOG_LEVELS {
-	INVALID_LEVEL = -1,
-	TRACE_MAX = 1,
-	TRACE_MED,
-	TRACE_MIN,
-	TRACE_PROTOCOL,
-	LOG_PROTOCOL = TRACE_PROTOCOL,
-	LOG_ERROR,
-	LOG_SEVERE,
-	LOG_FATAL,
-};
-
-extern "C" {
-	void Log(enum LOG_LEVELS, int, const char *, ...);
-}
-
-/////////////////////////////////////////////////////////////////////////////
 
 namespace mqtt {
 
@@ -110,8 +91,6 @@
 //
 void token::on_success(MQTTAsync_successData* rsp)
 {
-	::Log(TRACE_MIN, -1, "[cpp] on_success");
-
 	unique_lock g(lock_);
 	iaction_listener* listener = listener_;
 
@@ -154,8 +133,6 @@
 //
 void token::on_success5(MQTTAsync_successData5* rsp)
 {
-	::Log(TRACE_MIN, -1, "[cpp] on_success5");
-
 	unique_lock g(lock_);
 	iaction_listener* listener = listener_;
 	if (rsp) {
@@ -197,8 +174,6 @@
 //
 void token::on_failure(MQTTAsync_failureData* rsp)
 {
-	::Log(TRACE_MIN, -1, "[cpp] on_failure");
-
 	unique_lock g(lock_);
 	iaction_listener* listener = listener_;
 	if (rsp) {
@@ -230,8 +205,6 @@
 //
 void token::on_failure5(MQTTAsync_failureData5* rsp)
 {
-	::Log(TRACE_MIN, -1, "[cpp] on_failure5");
-
 	unique_lock g(lock_);
 	iaction_listener* listener = listener_;
 	if (rsp) {
